### PR TITLE
Add support for setting Global ID for Overviews from outside.

### PR DESCRIPTION
**Pull request by plyhun**
This functionality is required for importing.

### CODE REVIEW

File: cms-core/src/main/java/com/gentics/contentnode/rest/util/ModelBuilder.java [hunk 1]
```diff
@@ -735,6 +736,12 @@ public static com.gentics.contentnode.rest.model.Overview getOverview(
 			Overview nodeOverview) throws NodeException {
 		com.gentics.contentnode.rest.model.Overview restOverview = new com.gentics.contentnode.rest.model.Overview();
 
+		if (nodeOverview == null) {
+			return restOverview;
+		}
+
+		// Fill IDs
+		restOverview.setGlobalId(nodeOverview.getGlobalId().toString());
```
Comment (npomaroli) [inline cms-core/src/main/java/com/gentics/contentnode/rest/util/ModelBuilder.java:744]: Add null-check for nodeOverview.getGlobalId()

**Review by npomaroli — CHANGES_REQUESTED** [review]

File: cms-core/src/main/java/com/gentics/contentnode/rest/util/ModelBuilder.java [hunk 2]
```diff
@@ -1481,13 +1490,9 @@ public static Page getPage(com.gentics.contentnode.rest.model.Page restPage, boo
 			for (Iterator<com.gentics.contentnode.rest.model.Tag> i = restTags.values().iterator(); i.hasNext();) {
 				com.gentics.contentnode.rest.model.Tag restTag = i.next();
 				Tag tag = null;
```
Comment (plyhun) [inline cms-core/src/main/java/com/gentics/contentnode/rest/util/ModelBuilder.java:1492]: This alien set of changes is a consequence of a weird upstream merge. For instance, this change already exists in the target branch:
https://github.com/gentics/cms-oss/blob/hotfix-6.0.x/cms-core/src/main/java/com/gentics/contentnode/rest/util/ModelBuilder.java#L1484

**Comment by npomaroli** [comment]
Can you please make a clean MR out ot this? Based on hotfix-6.0.x and directed back to hotfix-6.0.x.

**Comment by plyhun** [comment]
--> https://github.com/gentics/cms-oss/pull/102

**Pull request closed without merging**